terraform/modules/vault/cert-manager.tf

```diff
@@ -1,10 +1,6 @@
 resource "vault_generic_secret" "cloudflare-api-key" {
-  for_each = toset(["chronos", "sandbox", "production"])
+  for_each = toset(["sandbox", "production"])
```

Comment:
Why do we still have sandbox here?

Reply:
I kept it there so we having the secrets set up for when (if) we want to spin up a sandbox cluster to test a major infra change